x/clp/keeper/calculations.go

```diff
@@ -286,6 +287,12 @@ func CalculatePoolUnits(oldPoolUnits, nativeAssetBalance, externalAssetBalance,
 		return sdk.ZeroUint(), sdk.ZeroUint(), types.ErrAsymmetricAdd
```

Comment:
Should the ratio change and ratioThreshold replace the old slipadjust check and symmetryThreshold?